linera-views/src/reentrant_collection_view.rs

```diff
@@ -326,6 +326,168 @@ where
     }
 }
 
+impl<C, W> ReentrantByteCollectionView<C, W>
+where
+    C: Context + Send + Clone + 'static,
+    ViewError: From<C::Error>,
+    W: View<C> + Send + Sync + 'static,
+{
+    /// Load multiple entries for write at once.
+    /// The entries in short_keys have to be all distincts.
+    /// ```rust
+    /// # tokio_test::block_on(async {
+    /// # use linera_views::memory::{create_test_context, MemoryContext};
+    /// # use linera_views::reentrant_collection_view::ReentrantByteCollectionView;
+    /// # use linera_views::register_view::RegisterView;
+    /// # use crate::linera_views::views::View;
+    /// # let context = create_test_context();
+    ///   let mut view : ReentrantByteCollectionView<_, RegisterView<_,String>> = ReentrantByteCollectionView::load(context).await.unwrap();
+    ///   let short_keys = vec![vec![0, 1], vec![2, 3]];
+    ///   let subviews = view.try_load_entries_mut(short_keys).await.unwrap();
+    ///   let value1 = subviews[0].get();
+    ///   let value2 = subviews[1].get();
+    ///   assert_eq!(*value1, String::default());
+    ///   assert_eq!(*value2, String::default());
+    /// # })
+    /// ```
+    pub async fn try_load_entries_mut(
+        &mut self,
+        short_keys: Vec<Vec<u8>>,
+    ) -> Result<Vec<OwnedRwLockWriteGuard<W>>, ViewError> {
+        let mut selected_short_keys = Vec::new();
+        let mut selected_to_clear = Vec::new();
+        *self.hash.get_mut() = None;
+        let updates = self.updates.get_mut();
+        for short_key in short_keys.clone() {
+            match updates.entry(short_key.clone()) {
+                btree_map::Entry::Occupied(entry) => {
+                    let entry = entry.into_mut();
+                    if let Update::Removed = entry {
+                        selected_short_keys.push(short_key);
+                        selected_to_clear.push(true);
```

Comment:
Seems like we could just use one vector: `selected_short_keys.push((short_key, /* should clear */ true));`

Reply:
Actually, I have some ideas to remove cloning and make `short_keys` an `Impl IntoIterator` but maybe this can wait a bit.

Reply:
Of course.